test_scripts/AppIconsFolder/ATF_P_AppIconsFolder_exists_no_ReadWritePermissions.lua

```diff
@@ -0,0 +1,143 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+--    [GENIVI] Conditions for SDL to create and use 'AppIconsFolder' storage 
+--    [AppIconsFolder]: Folder defined at "AppIconsFolder" param has NO read-write permissions
+-- Description:
+--    SDL behavior when on startup AppIconsFolder exists but has no permissions
+-- 1. Used preconditions:
+--      Stop SDL
+--      Set AppIconsFolder in .ini file
+--      Start SDL and HMI
+--      Connect mobile
+-- 2. Performed steps:
+--      Register app
+--      Send SetAppIcon 
+-- Expected result:
+--    SDL must not use the 'store-app's-icon' mechanism
+---------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_resumption')
+require('cardinalities')
+local mobile_session = require('mobile_session')
+
+--[[ Required Shared Libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+require('user_modules/AppTypes')
+
+--[[ Local variables ]]
+local RAIParameters = config.application1.registerAppInterfaceParams
+local folderNoPermissions = commonPreconditions:GetPathToSDL() .. "FolderWithoutPermissions"
+
+--[[ General Precondition before ATF start ]]
+commonSteps:DeleteLogsFileAndPolicyTable()
+assert(os.execute( "rm -rf " .. folderNoPermissions))
+commonFunctions:SetValuesInIniFile("AppIconsFolder%s-=%s-.-%s-\n", "AppIconsFolder", 'FolderWithoutPermissions')
+
+--[[ Local functions ]]
+local function registerApplication(self)
+  local corIdRAI = self.mobileSession:SendRPC("RegisterAppInterface", RAIParameters)
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered",
+  {
+    application =
+    {
+     appName = RAIParameters.appName
+    }
+  })
+  :Do(function(_,data)
+    self.applications[RAIParameters.appName] = data.params.application.appID
+  end)
+  self.mobileSession:ExpectResponse(corIdRAI, { success = true, resultCode = "SUCCESS" })
+end
+
+local function pathToAppFolderFunction(appID)
+  return folderNoPermissions .. "/" .. appID .. "_" .. config.deviceMAC .. "/"
```

Comment:
@aarotar, It seems correct path has to be the following, like in other scripts:
`commonPreconditions:GetPathToSDL() .. "storage/" .. appID .. "_" .. config.deviceMAC .. "/"`
But currently it will be:
`commonPreconditions:GetPathToSDL() .. "FolderWithoutPermissions" .. "/" .. appID .. "_" .. config.deviceMAC .. "/"`

'FolderWithoutPermissions' folder will be used instead of 'storage'